test/lib/fixtures.ts

```diff
@@ -172,7 +172,6 @@ export class NetworkFixture {
     await staking.connect(deployer).setSlasher(slasherAddress, true)
     await gns.connect(deployer).approveAll()
     if (isL2) {
-      await grt.connect(deployer).addMinter(l2GraphTokenGateway.address)
```

Comment:
Can you remind me what we removed the L2 bridge to be a minter in the fixture?

Reply:
Because we are already using the `L2GraphToken.setGateway` function to add it as the gateway minter, that can mint and burn through the bridgeMint and bridgeBurn functions.